### PR TITLE
Add verification step to image release process

### PR DESCRIPTION
**Pull request by potiuk**
---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragement file, named `{pr_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

File: dev/README_RELEASE_AIRFLOW.md [hunk 1]
```diff
@@ -1041,6 +1042,16 @@ Preparing a release that is not in the latest branch:
...
 ```
 
+## Verify production images
+
+```shell script
+for PYTHON in 3.7 3.8 3.9 3.10
+do
+    docker pull apache/airflow:${VERSION}-python${PYTHON}
+    breeze verify-prod-image --image-name apache/airflow:${VERSION}-python${PYTHON}
+done
```
Comment (jedcunningham) [inline dev/README_RELEASE_AIRFLOW.md:1052]:
```suggestion
done
docker pull apache/airflow:${VERSION}
breeze verify-prod-image --image-name apache/airflow:${VERSION}
```

Also worth verifying the bare version tag?

Comment (potiuk) [reply dev/README_RELEASE_AIRFLOW.md:1052]: This is just an alias. Makes no sense to run tests for it I think.

Comment (jedcunningham) [reply dev/README_RELEASE_AIRFLOW.md:1052]: Should at least pull then to make sure the tag was pushed?

**Review by jedcunningham — APPROVED** [review]

File: dev/README_RELEASE_AIRFLOW.md (hunk 1 above)
Comment (potiuk) [reply dev/README_RELEASE_AIRFLOW.md:1052]: Why not :)

**Pull request merged**